src/views/local-token-service.handlebars

```diff
@@ -123,6 +123,15 @@ let text = client.get("<span class="origin">https://gateway.local</span>/things"
         </div>
       </div>
     </section>
+    <script>
+/* postToken to client apps if any */
+if (config.get('oauthPostToken') && (window.location.protocol === "file:") {
```

Comment:
The Content-Security-Policy setting means we'll only be embedded in trusted situations so this doesn't need the config.get here

Reply:
I still use oauthPostToken to prevent a not wanted post action 
see next revison